### ISSUE TITLE
[Feature Request] Provide a way to write optional code hide in comment

### ISSUE THREAD

**Issue by loynoir** [nippur72/ifdef-loader#45]
### Background
Sometime, mix all conditional code together may not allowed by linters.
Need a way to write optional code, able to avoid linting errors.

### Actual
```js
/// #if ! __DEBUG__
const foo = "bar" // Cannot redeclare block-scoped variable 'foo'
/// #else
var foo = "bar" // Cannot redeclare block-scoped variable 'foo'
const __hotUpdate = ()=>{
  foo="bar2" // Cannot assign to 'foo' because it is a constant
}
/// #endif
```

### Expected
```js
/// #if ! __DEBUG__
const foo = "bar"
/// #else
/// var foo = "bar"
/// const __hotUpdate = ()=>{
///   foo="bar2"
/// }
/// #endif
```

### Maybe?
1.  `// code`
2.  `/// code`
3.  `/* code */`

**Comment by nippur72** [comment]
what about
```js
/// #if ! __DEBUG__
const foo = "bar"
/// #else
//_ var foo = "bar"
//_ const __hotUpdate = ()=>{
//_   foo="bar2"
//_ }
/// #endif

```

**Comment by loynoir** [comment]
Thanks for reply.
I think it's better. 
Because it's easier to find out there is a different block within a glance.

**Comment by loynoir** [comment]
Thanks for your reply.
I think it's better. 
Because it's easier to find out there is a different block within a glance.

**Comment by nippur72** [comment]
this is now addressed in v2.3.0 with the option `ifdef-uncomment-prefix` (see the README).
